packages/requisitions/src/RequestRequisition/api/hooks.ts

```diff
@@ -320,6 +321,10 @@ export const useDeleteRequestLines = () => {
   );
 
   const onDelete = async () => {
+    if (isDisabled) {
+      info(t('label.cant-delete-disabled'))();
```

Comment:
```suggestion
      info(t('label.cant-delete-disabled-requisition'))();
```

the key `label.cant-delete-disabled` isn't in `replenishment` namespace

Reply:
Ah, I knew I'd miss one! Thanks.